root/static/styles/forms.less

```diff
@@ -751,3 +752,47 @@ div.rel-editor-dialog {
 button.guess-timezone {
     background-image: data-uri('../images/icons/magnet.png');
 }
+
+.popper-dialog {
+    visibility: hidden;
+    padding: 0.5em;
+    position: absolute;
+    max-width: 500px;
+    background: @text-white;
+    border: 1px solid #AAA;
+    top: 0;
+    left: 0;
```

Comment:
For the example here, at least, a bit higher than 0 would seem good, so that it doesn't cover the icon but appears to come just to the right of it.